doc-content/enterprise-only/release-notes/rn-740-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,126 @@
+[id='rn-740-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.4.0
+
+{PRODUCT} 7.4.0 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+[NOTE]
+====
+{PRODUCT_DM} fixed issues apply to {PRODUCT_PAM} as well. For a list of {PRODUCT_DM} 7.3.0 fixed issues, see the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.3/html-single/release_notes_for_red_hat_decision_manager_7.3/index[_Release Notes for Red Hat Decision Manager 7.3_]
+====
+
+== {CENTRAL}
+* Ability to set the Smart Router location to an external URL [https://issues.jboss.org/browse/RHPAM-1387[RHPAM-1387]]
+* Asset search does not find assets with an extension when using ElasticSearch/Infinispan indexing [https://issues.jboss.org/browse/RHPAM-1249[RHPAM-1249]]
+* Assets created on freshly forked branches appear in their parent branches in a {CENTRAL} cluster [https://issues.jboss.org/browse/RHPAM-1821[RHPAM-1821]]
+* {CENTRAL} is unable to connect to a co-located controller when deployed at the same time [https://issues.jboss.org/browse/RHPAM-1910[RHPAM-1910]]
+* {CENTRAL} Monitoring fails to start a new process in an unmanaged immutable {KIE_SERVER} [https://issues.jboss.org/browse/RHPAM-2101[RHPAM-2101]]
+* The spaces breadcrumb is hidden [https://issues.jboss.org/browse/RHPAM-2091[RHPAM-2091]]
+* Drop-downs in the Security perspective can not be controlled by the keyboard [https://issues.jboss.org/browse/RHPAM-2085[RHPAM-2085]]
+* Error for process diagram containing multiple instance user tasks [https://issues.jboss.org/browse/RHPAM-2020[RHPAM-2020]]
+* Guided decision able is not getting updated [https://issues.jboss.org/browse/RHPAM-2007[RHPAM-2007]]
+* HTTP 503 error is shown in the Monitoring console when {KIE_SERVER} is scaled to 0 [https://issues.jboss.org/browse/RHPAM-2008[RHPAM-2008]]
+* Installing a service task from a project *Settings* tab only updates the master branch [https://issues.jboss.org/browse/RHPAM-2026[RHPAM-2026]]
+* Missing translations in the UI [https://issues.jboss.org/browse/RHPAM-1549[RHPAM-1549]]
+* Not able to build a project that has a guided scorecard [https://issues.jboss.org/browse/RHPAM-2055[RHPAM-2055]]
+* Problem expanding or dragging the right-most column in the data table of a guided rule template [https://issues.jboss.org/browse/RHPAM-1559[RHPAM-1559]]
+//* Process diagram is not displayed after importing a project [https://issues.jboss.org/browse/RHPAM-1881[RHPAM-1881]]
```

Comment:
is this commented out? RHPAM-1881 has been verified for 7.4.0

Reply:
Uncommented